Doc/whatsnew/3.9.rst

```diff
@@ -371,6 +371,12 @@ finalization crashed with a Python fatal error if a daemon thread was still
 running.
 (Contributed by Victor Stinner in :issue:`37266`.)
 
+time
+----
+
+:func:`~time.thread_time` now output in nanosecond resolution via
+``thread_cputime`` on AIX. (Contributed by Batuhan Taskaya in :issue:`40192`)
```

Comment:
Oh, I proposed a different text, but it seems like my comment is lost :-( I propose:

```
On AIX, :func:`~time.thread_time` is now implemented with ``thread_cputime()``
which has nanosecond resolution, than than
``clock_gettime(CLOCK_THREAD_CPUTIME_ID)`` which has a resolution of 10 ms.
(Contributed by Batuhan Taskaya in :issue:`40192`)
```

Use the same text in the NEWS entry, but replace "(Contributed by Batuhan Taskaya in :issue:`40192`)" with "Patch by Batuhan Taskaya.".